### DISCUSSION TITLE
Drifty Capture for Windows Machine

### DISCUSSION THREAD

**Discussion by hasnainmakada-99** [SaptarshiSarkar12/Drifty#354] [Q&A] [answered]
Hey @SaptarshiSarkar12 Can I include this windows capture where I explain how to download the drifty GUI and use it inside the windows machine

Here's the video

https://github.com/SaptarshiSarkar12/Drifty/assets/82728823/38b2fff4-96b5-46b1-ad35-0721feedb108

**Comment by SaptarshiSarkar12** [comment, marked as answer]
@hasnainmakada-99 Yes, you can include that. Please trim the video from **0:25** and fast forward the downloading part (from 0:54).
I would suggest you to record the whole portion in two parts -
- First one will be that you are going to download the file (You can **only** share the browser window to OBS and it will record what's happening in the browser window.) You should also **fast forward** the portion of the video in which you are showing the progress of the download of the executable.
- Second one will be opening the app and showing the usage of the Application.  

At last, you can merge the two - and tada :tada:! It's done :partying_face:!

**Reply by hasnainmakada-99** [reply]
Yeah sure @SaptarshiSarkar12 I will surely do that and will update you by tonight

**Reply by SaptarshiSarkar12** [reply]
Thank you :grin: @hasnainmakada-99!

**Reply by SaptarshiSarkar12** [reply]
@hasnainmakada-99 If you do not have any further question on this topic, you can close this discussion :smile:.

**Reply by hasnainmakada-99** [reply]
Let's keep it open for sometime @SaptarshiSarkar12 because as I'll proceed I'll need to discuss things with you

**Reply by SaptarshiSarkar12** [reply]
@hasnainmakada-99 Okay. No problem.

**Reply by hasnainmakada-99** [reply]
@SaptarshiSarkar12 Let me know your views on this, It's now been edited

https://drive.google.com/file/d/1ce7Kn5uApeGz4ViNd1HMQg_N8ik9juN0/view?usp=sharing

**Reply by SaptarshiSarkar12** [reply]
@hasnainmakada-99 Looks good. You can add it.

**Comment by hasnainmakada-99** [comment]
Hey @SaptarshiSarkar12 I need your help brother,

I am currently done with the getting started and quick Start guide

and I am going to work on the FaQs readme, so I would like to know what are some of the most common questions which the community has and will you help me curate that

**Reply by SaptarshiSarkar12** [reply]
@hasnainmakada-99 The most asked question in the community is how to setup the project (both Website and Java code in IDE) in their local environment. I think this can be included in other sections because they are going to be a lengthy guide. What do you think?

Some questions I remember are :point_down: 
1. **Why does Drifty GUI not open in my Windows machine?**
:arrow_right: Your Windows machine needs to have **Microsoft Visual C++ Redistributable** installed on your machine. You can download and install it from [its official download page](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170).

2. **Which IDE should I use for opening Drifty App code in my local environment?**
:arrow_right: You can use any IDE you like but, it is recommended to use [IntelliJ Idea](https://jetbrains.com/idea). You can use the free **Community** version of IntelliJ Idea if you do not have a JetBrains subscription.

3. **How can I get updates on this project?**
:arrow_right: You can join our [Discord server](https://discord.gg/DeT4jXPfkG) to get updates on the project. You can also watch this repository by clicking on the **watch** button [here](https://github.com/SaptarshiSarkar12/Drifty) and select the notification types to **All Activity** to get all updates on this project.

4. **What technologies does Drifty use?**
:arrow_right: The technologies that Drifty uses are :point_down:
- **Java** - for Drifty Application (both CLI and GUI)
- **JavaFX** - for Drifty's GUI part only
- **Docker** - for containerizing the Application to provide public access to beta features
- **Next.Js** -  for building [the Drifty's Website](https://saptarshisarkar12.github.io/Drifty)

5. **How can I contribute to Drifty?**
:arrow_right: You can check the **Contribution** section of this docs to learn how to contribute to this project.

6. **How can I update Drifty to the latest version?**
:arrow_right: Currently, automatic updates are not supported and is a [Work-In-Progress](https://github.com/SaptarshiSarkar12/Drifty/issues/217). So, you need to download [the latest executable from the website](https://saptarshisarkar12.github.io/Drifty/download) and install it accordingly.

7. **How can I access the Beta features in Drifty?**
:arrow_right: Currently, Beta features are available only through Docker images. You can check the docs (@hasnainmakada-99 the link for this need to be added). Please provide feedback on them by creating a new Discussion [here](https://github.com/SaptarshiSarkar12/Drifty/discussions).

**Reply by SaptarshiSarkar12** [reply]
@hasnainmakada-99 If you think any other questions can be included, please let me know.
